src/main/java/TaskManager.java

```diff
@@ -0,0 +1,48 @@
+public class TaskManager {
+    private final Task[] taskList;
+    private int currentTaskIndex;
+
+    public TaskManager(){
+        currentTaskIndex = 0;
+        taskList = new Task[100];
```

Comment:
Should the maximum number of tasks be a constant instead of the magic number 100?